hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/CacheConfig.java

```diff
@@ -93,6 +93,8 @@ public class CacheConfig {
   public static final String DROP_BEHIND_CACHE_COMPACTION_KEY =
     "hbase.hfile.drop.behind.compaction";
 
+  public static String PREFETCH_PERSISTENCE_PATH_KEY = "hbase.prefetch.persist.filepath";
```

Comment:
nit: So this is basically a path for a list of prefetched files. Can we call it `hbase.prefetch.file-list.path`? That would be more intuitive, especially when we start to use it in BucketCache class, which already has a `persistencePath` variable. We could then relabel `BuckeCache.prefetchPersistencePath` to something like `BuckeCache.prefetchedFileListPath`.